src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,18 @@
-export const TodoList = () => {};
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
```

Comment:
Consider moving the `Todo` type to a separate file for better organization and reusability. 📁